source/utils/ConfigManager.h

```diff
@@ -26,12 +26,21 @@
 class CreatureDefinition;
 class Weapon;
 class SpawnCondition;
+class CreatureMood;
 
 //! \brief This class is used to manage global configuration such as network configuration, global creature stats, ...
 //! It should NOT be used to load level specific stuff. For that, there if GameMap.
 class ConfigManager : public Ogre::Singleton<ConfigManager>
 {
 public:
+    enum CreatureMoodLevel
+    {
+        Happy,
+        Neutral,
+        Upset,
+        Angry,
+        Furious
+    };
```

Comment:
> Let's switch "when we see one in our way"

Yes, I was thinking about the same as for nullptr: when we see one. No need to rush.